src/legacy/server/logging/log_format_json.test.js

```diff
@@ -97,6 +99,79 @@ describe('KbnLoggerJsonFormat', () => {
       expect(message).toBe('memory: 0.0B uptime: 0:00:00 load: [1.00 1.00 2.00] delay: 0.000');
     });
 
+    describe('with metadata', () => {
+      it('logs an event with meta data', async () => {
+        const event = {
+          data: attachMetaData('message for event', {
+            prop1: 'value1',
+            prop2: 'value2',
+          }),
+          tags: ['tag1', 'tag2'],
+        };
+        const result = await createPromiseFromStreams([createListStream([event]), format]);
+        const { level, message, prop1, prop2, tags, } = JSON.parse(result);
+
+        expect(level).toBe(undefined);
+        expect(message).toBe('message for event');
+        expect(prop1).toBe('value1');
+        expect(prop2).toBe('value2');
+        expect(tags).toEqual(['tag1', 'tag2']);
+      });
+
+      it('meta data rewrites event fields', async () => {
+        const event = {
+          data: attachMetaData('message for event', {
+            tags: ['meta-data-tag'],
```

Comment:
not sure it's desired behaviour, but we have to keep it for BWC

Reply:
Should this even be possible? It seems we don't include the `tags` field in the metadata object anyways?

Reply:
`tags` is just one example. other possible collisions:
https://github.com/restrry/kibana/blob/97f4567ae9d912b1ec02942328bffe86a9d12b3b/src/legacy/server/logging/log_format.js#L80-L83

Reply:
> Should this even be possible? It seems we don't include the tags field in the metadata object anyways?

It's possible in LP. as this test shows 😅

Reply:
As far as I understand it's possible only until we drop legacy logging entirely. In the NP logging we'll only have `meta`  and not dedicated `tags`, right?

Reply:
It depends on the log formats in NP. I'd avoid such weird things.